<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>830</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>830</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DaRK Calories Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/images/icon.svg</normaloff>:/images/images/icon.svg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">   *:focus {
    	border: none;
    	outline: none;
    }

QGroupBox { 
     border: none; 
 } 

QLineEdit{
color:white;
}</string>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>830</width>
     <height>560</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>830</width>
     <height>560</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>830</y>
      <width>560</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="bgLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>830</width>
      <height>560</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/images/background.svg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/images/background.svg</pixmap>
    </property>
   </widget>
   <widget class="QGroupBox" name="genderBox">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>130</y>
      <width>267</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton{
color:white;
	font: 12pt &quot;Rubik&quot;;
}

QRadioButton::indicator {
    width: 16px;
    height: 16px;
}

QRadioButton::indicator::unchecked {
    
	image: url(:/images/images/radio_unchecked);
	

}

QRadioButton::indicator:unchecked:hover {
    
	image: url(:/images/images/radio_unchecked_hover);
}

QRadioButton::indicator:unchecked:pressed {
    
	image: url(:/images/images/radio_unchecked_pressed);
}

QRadioButton::indicator::checked {
    
	image: url(:/images/images/radio_selected.svg);
}

QRadioButton::indicator:checked:hover {
	image: url(:/images/images/radio_selected_hover.svg);
}

QRadioButton::indicator:checked:pressed {
	
	image: url(:/images/images/radio_selected_pressed.svg);
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>78</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rubik</family>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>Gender</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="maleRB">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Male</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="femaleRB">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Female</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="hightBox">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>198</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>78</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rubik</family>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>Hight</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hightInput">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>119</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background-color:rgba(255, 255, 255, 0);
border:none;
border-bottom: 2px solid white;
border-radius:2px;
}</string>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>25</y>
       <width>24</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font: 12pt &quot;Rubik&quot;;</string>
     </property>
     <property name="text">
      <string>CM</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="weightBox">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>266</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>78</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rubik</family>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>Weight</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="weightInput">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>119</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background-color:rgba(255, 255, 255, 0);
border:none;
border-bottom: 2px solid white;
border-radius:2px;
}</string>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>25</y>
       <width>24</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font: 12pt &quot;Rubik&quot;;</string>
     </property>
     <property name="text">
      <string>KG</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ageBox">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>334</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>78</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rubik</family>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>Age</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ageInput">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>100</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background-color:rgba(255, 255, 255, 0);
border:none;
border-bottom: 2px solid white;
border-radius:2px;
}</string>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>25</y>
       <width>41</width>
       <height>18</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font: 12pt &quot;Rubik&quot;;</string>
     </property>
     <property name="text">
      <string>Years</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="activityBox">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>402</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rubik</family>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>Activity</string>
     </property>
    </widget>
    <widget class="QComboBox" name="activityCombo">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>155</width>
       <height>29</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox
{
    selection-background-color: #379ec4;
    border: 1px solid #1e1e1e;
    border-radius: 5;
    border: none;
    border-bottom: 2px solid white;
    border-radius: 2px;
    color:white;
    min-width: 6em;
    background-color:rgba(255, 255, 255, 0);
}

QComboBox:hover,QPushButton:hover
{
    border-bottom: 2px solid white;
    border-radius: 2px;
}


QComboBox:on
{
    padding-top: 3px;
    padding-left: 4px;
    selection-background-color: #379ec4;
}

QComboBox QAbstractItemView
{
    selection-background-color: #379ec4;
}


QComboBox::down-arrow
{
    
	image: url(:/images/images/arrow.svg);
} 
</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>Little /no exercise</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Exercise 1-2 times/week</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Exercise 2-3 times/week</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Exercise 4-5 times/week</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Exercise 6-7 times/week</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Professional athlete</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="resultBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>240</y>
      <width>190</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:white;
font: 12pt &quot;Rubik&quot;;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <widget class="QLabel" name="resultLabel1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>125</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="resultLabel2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>35</y>
       <width>125</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="resultLabel3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>125</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="resultLabel4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>105</y>
       <width>125</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color:#D4D4D4;
font: 13pt &quot;Rubik&quot;;
border:none;
background-color:rgba(255, 255, 255, 0);
}

QPushButton:hover{
color:#192125;
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calcBtn">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>410</y>
      <width>125</width>
      <height>38</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgba(255, 255, 255, 1);
border:2px solid white;
border-top-left-radius: 5px;
border-bottom-left-radius: 5px;
color:#056082;
font: 11pt &quot;Rubik&quot;;
padding-bottom:2px;
}

QPushButton:hover{
border:rgba(255, 255, 255, 0.9);
background-color:rgba(255, 255, 255, 0.9);
}</string>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorMessage">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>470</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="aboutButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color:#D4D4D4;
font: 20PX &quot;Rubik&quot;;
border:none;
background-color:rgba(255, 255, 255, 0);
}

QPushButton:hover{
color:#192125;
}</string>
    </property>
    <property name="text">
     <string>!</string>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="clearBtn">
    <property name="geometry">
     <rect>
      <x>651</x>
      <y>410</y>
      <width>65</width>
      <height>38</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgba(255, 255, 255, 1);
border:2px solid white;
border-top-right-radius: 5px;
border-bottom-right-radius: 5px;
color:#056082;
font: 11pt &quot;Rubik&quot;;
padding-bottom:2px;
}

QPushButton:hover{
border:rgba(255, 255, 255, 0.9);
background-color:rgba(255, 255, 255, 0.9);
}</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <zorder>bgLabel</zorder>
   <zorder>widget</zorder>
   <zorder>genderBox</zorder>
   <zorder>hightBox</zorder>
   <zorder>weightBox</zorder>
   <zorder>ageBox</zorder>
   <zorder>activityBox</zorder>
   <zorder>resultBox</zorder>
   <zorder>exitButton</zorder>
   <zorder>calcBtn</zorder>
   <zorder>errorMessage</zorder>
   <zorder>aboutButton</zorder>
   <zorder>clearBtn</zorder>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
